Code/Mantid/MantidPlot/src/Mantid/MantidMatrix.cpp

```diff
@@ -64,6 +65,53 @@ namespace
   Logger g_log("MantidMatrix");
 }
 
+
+namespace {
+/**
+ * Converts an interger value to the corrsponding enum
+ * @param i: the integer to check
+ * @returns the corresponding model type
+ */
+MantidMatrixModel::Type intToModelType(int i) {
+  switch(i) {
+    case 0:
+      return MantidMatrixModel::Y;
+    case 1:
+      return MantidMatrixModel::X;
+    case 2:
+      return MantidMatrixModel::E;
+    case 3:
+      return MantidMatrixModel::DX;
+    default:
+      g_log.error("Trying to convert to an unknown MantidMatrixModel type. Defaulting to Y type.");
+      return MantidMatrixModel::Y;
+  }
+}
+
+/**
+ * Converts an model enum into a corresponding integer
+ * @parm type: the model type to check
+ * @returns the corresponding integer value
+ */
+int modelTypeToInt(MantidMatrixModel::Type type) {
```

Comment:
For future. I would consider putting unit tests over these free functions and adding a comment to those unit tests saying something like "If you're changing these mappings think about it very hard".